cli/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 rust-version = "1.60"
 
 [features]
-default = ["selfmanage"]
+default = ["selfmanage", "extensions"]
```

Comment:
When is the next CLI release planned? Do we want to enable extensions already?

Reply:
Just for my personal tests because I was getting sick of `--all-features`.